packages/strapi-design-system/src/Switch/Switch.stories.mdx

```diff
@@ -36,7 +36,7 @@ Activated state is one of two of them available for a Switch.
 
 <Canvas>
   <Story name="activated">
-    <Switch label="Activate the microphone" selected={true} onChange={() => setChecked((s) => !s)} visibleLabels />
+    <Switch label="Activate the microphone" selected={true} onChange={(e) => console.log(e)} visibleLabels />
```

Comment:
I guess these are a mistake?

Reply:
Well, they're technically `undefined` so the state didn't exist in the first place for the story. Could add it correctly if you prefer?

Reply:
Yeah, I think I'd prefer if the stories would work correctly.